From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 9 Nov 2021 20:40:06 +0000
Subject: Skip flaky tests

---
 nbclient/tests/test_client.py | 5 -----
 1 file changed, 5 deletions(-)

--- a/nbclient/tests/test_client.py
+++ b/nbclient/tests/test_client.py
@@ -320,16 +320,6 @@
         ("Factorials.ipynb", {"kernel_name": "python"}),
         ("HelloWorld.ipynb", {"kernel_name": "python"}),
         ("Inline Image.ipynb", {"kernel_name": "python"}),
-        (
-            "Interrupt.ipynb",
-            {
-                "kernel_name": "python",
-                "timeout": 3,
-                "interrupt_on_timeout": True,
-                "allow_errors": True,
-            },
-        ),
-        ("Skip Exceptions.ipynb", {"kernel_name": "python", "allow_errors": True}),
         ("Skip Execution with Cell Tag.ipynb", {"kernel_name": "python"}),
         ("SVG.ipynb", {"kernel_name": "python"}),
         ("Unicode.ipynb", {"kernel_name": "python"}),
