From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 9 Nov 2021 20:40:06 +0000
Subject: Skip flaky tests

---
 nbclient/tests/test_client.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/nbclient/tests/test_client.py b/nbclient/tests/test_client.py
index 2abaa82..4b393aa 100644
--- a/nbclient/tests/test_client.py
+++ b/nbclient/tests/test_client.py
@@ -258,11 +258,6 @@ def filter_messages_on_error_output(err_output):
         ("Factorials.ipynb", dict(kernel_name="python")),
         ("HelloWorld.ipynb", dict(kernel_name="python")),
         ("Inline Image.ipynb", dict(kernel_name="python")),
-        (
-            "Interrupt.ipynb",
-            dict(kernel_name="python", timeout=1, interrupt_on_timeout=True, allow_errors=True),
-        ),
-        ("Skip Exceptions.ipynb", dict(kernel_name="python", allow_errors=True)),
         ("Skip Execution with Cell Tag.ipynb", dict(kernel_name="python")),
         ("SVG.ipynb", dict(kernel_name="python")),
         ("Unicode.ipynb", dict(kernel_name="python")),
